### PR TITLE
[ITBL-5303] Update documentation with recent SDK changes

### PR DESCRIPTION
**Pull request by vbabenkoru**
\+ migration notes

### CODE REVIEW

**Comment by steelbrain** [comment]
These documents only specify `onCreate` for Deep Linking, `onCreate` is only invoked when the app is previously closed and is started to fulfill the intent.

It should also be documented that when the app is already running and the user clicks an App recognized URL, `onNewIntent` will be invoked instead.

**Comment by vbabenkoru** [comment]
@steelbrain It is covered here:
https://github.com/Iterable/iterable-android-sdk/blob/feature/ITBL-5303-sdk-updates-docs/README.md#handling-email-links

**Comment by steelbrain** [comment]
I somehow missed that 🤦‍♂️ , thanks for the pointer!

File: README.md [hunk 1]
```diff
@@ -1,53 +1,168 @@
...
+
+If you already have a `urlDelegate` (see *Handling links from push notifications* section above), the same handler can be used for email deep links by calling `handleAppLink` in the activity that handles all app links in your app:
+
+```java
+// MainActivity.java
+@Override
+public void onCreate() {
+	super.onCreate();
+	...
+	hanldeIntent(getIntent());
```
Comment (steelbrain) [inline README.md]: Typo, should be `handleIntent`

Comment (vbabenkoru) [reply README.md]: Fixed, thanks!

**Comment by steelbrain** [comment]
Hi @vbabenkoru 

The `AndroidManifest.xml` part is out-of-date at https://support.iterable.com/hc/en-us/articles/115000331943-Setting-up-Android-Push-Notifications (`IterablePushReceiver` no longer exists)

Any plans to update that page or move that doc part to the repository?

**Comment by davidtruong** [comment]
@steelbrain: We are currently working on updating the support docs.

**Pull request merged**